src/internal/testutil/cmds.go

```diff
@@ -166,7 +166,7 @@ func PachctlBashCmdCtx(ctx context.Context, t *testing.T, c *client.APIClient, s
 	// If you're wondering why you're not seeing stderr, it's probably because you got here via
 	// PachctlBashCmd (without a context), and thus there is no logger to log to.  Supply
 	// pctx.TestContext(t) as context, and you'll be good to go.
-	stderr := log.WriterAt(ctx, log.DebugLevel)
+	stderr := log.WriterAt(pctx.TestContext(t), log.DebugLevel)
```

Comment:
Hey, does this need to be reverted? I think the comment above implies that pachctlbashcmd intentionally logs without showing stderr.

Reply:
Got it. I'll remove that. I remember in another test this pct.TestContext was used, so I thought this might be helpful to keep.